CONTRIBUTING.md

```diff
@@ -22,6 +23,11 @@ __Example:__
   },
   "github": {
     "personalAccessToken": "TO_DEFINE_BY_YOU"
+  },
+  "logger": {
```

Comment:
This is not really related to the logger.
This is related to Discord for now, but globally, for the profile.
So maybe you should just add this option directly on the root of the env instead of the root of the env logger.

Reply:
"So maybe you should just add this option directly on the root of the env instead of the root of the env logger.".
I did not mentioned app ;)